client/src/pages/presentational/PollCard.jsx

```diff
@@ -42,9 +47,7 @@ const PollCard = ({ question, images, pollId, isUser = false, deletePoll }) => {
         <Button>
           <DeleteForever onClick={() => deletePoll(pollId)} />
         </Button>
-      ) : (
-        <></>
-      )}
```

Comment:
this button should only appear if it's the user

Reply:
hmm ya all I did was change it to `null` so it'd look neat. any diff no?

Reply:
just saw the fix